test/utils/test_pulse_scaling.py

```diff
@@ -138,3 +138,33 @@ def test_forced_rzz_template_match(self):
                 Operator(scale_qc_match.bind_parameters({theta: theta_set}))
             )
         )
+
+    def test_angle_reduction(self):
+        """Test Angle Reduction"""
```

Comment:
This test should take a different approach. It should directly instantiate a ReduceAngles pass, run the pass on a circuit with the gate, and check that the resultant gate has a reduced angle. That's all it needs to do.

Reply:
Rewrote pass to only check that angle is reduced.